### DISCUSSION TITLE
Option to add hcxtools and Wifite2?

### DISCUSSION THREAD

**Discussion by SgtFoose** [adde88/openwrt-useful-tools#17] [Ideas]
Thanks for you Git! I have a question, would it be possible to add Wifite2?

[https://github.com/derv82/wifite2](url)

**Comment by adde88** [comment]
As a package? Sure! Totally doable. 😄 
I actually made a package for the old MK5 some years back.
Do you think there's a high request for it though? Considering it's as easy to install Wifite2 on the Pineapples as on the Desktop PC. 🤔

**Reply by SgtFoose** [reply]
Yes sure, Pineapple is used by allot of new to hacking users. If you build it, I will make a video on YouTube with credits of course !
